src/en/tcbscans/src/eu/kanade/tachiyomi/extension/en/tcbscans/TCBScans.kt

```diff
@@ -25,24 +27,38 @@ class TCBScans : ParsedHttpSource() {
     override val lang = "en"
     override val supportsLatest = false
     override val client: OkHttpClient = network.cloudflareClient
-
+    val migrateMessage = "Migrate from TCB Scans to TCB Scans"
```

Comment:
Would be better to create it as a constant.

```suggestion
    companion object {
        private const val MIGRATE_MESSAGE = "Migrate from TCB Scans to TCB Scans"
    }
```

And then replace all occurrences of `migrateMessage` with `MIGRATE_MESSAGE`.